mobile/library/objective-c/EnvoyEventTracker.h

```diff
@@ -0,0 +1,15 @@
+#import <Foundation/Foundation.h>
+
+#import "library/objective-c/EnvoyAliases.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface EnvoyEventTracker : NSObject
+
+@property (nonatomic, copy, nonnull) void (^track)(EnvoyEvent *);
+
+- (instancetype)initWithEventTrackingClosure:(nonnull void (^)(EnvoyEvent *))track;
```

Comment:
nit picking: not sure whether `nonnull` attributes in here and at line #9 are needed but it was probably just copied-over and it is not a problem.

Reply:
Good eye! Removed.